motionSetPointsApp/src/setPoint.cpp

```diff
@@ -19,10 +26,32 @@
 
 #define ROW_LEN 200
 
+class FileIO : public FileIOInterface {
+public:
+    virtual void Open(const char* filename) {
+        file.open(filename);
+    }
+
+    virtual bool ReadLine(std::string &str) {
+        return (bool)std::getline(file, str);
+    }
+
+    virtual void Close() {
+        file.close();
+    }
+
+    virtual bool isOpen() {
+        return file.is_open();
+    }
+
+private:
+    std::ifstream file;
+};
```

Comment:
I think file_ or m_file is better as it is a class variable, though we should have our own code convention.